api/app/domain/analyzers/dummy_tree_cover_analyzer.py

```diff
@@ -0,0 +1,16 @@
+from random import random
+
+from app.domain.analyzers.analyzer import Analyzer
+from app.domain.models.analysis import Analysis
+from app.models.land_change.tree_cover import TreeCoverAnalyticsIn
+
+
+class DummyTreeCoverAnalyzer(Analyzer):
+    async def analyze(self, analysis: Analysis):
+        land_cover_change_analytics_in = TreeCoverAnalyticsIn(**analysis.metadata)
+
+        return {
+            "aoi_id": analysis.metadata["aoi"]["ids"] ,
+            "aoi_type": [land_cover_change_analytics_in.aoi.type],
+            "area_ha": random() * 100,
```

Comment:
`aoi_type` and `area_ha` have to be a list of length `metadata.aoi.ids`

Reply:
Roger, thanks!